doc/modules/model_evaluation.rst

```diff
@@ -1353,6 +1393,21 @@ to the given limit.
     for an example of using ROC to
     model species distribution.
 
+  * See :ref:`sphx_glr_auto_examples_model_selection_plot_roc.py`
+    for an example of using multiclass one-vs-one ROC.
+
+.. topic:: References:
+
+    .. [HT2001] Hand, D.J. and Till, R.J., 2001. `A simple generalisation
+       of the area under the ROC curve for multiple class classification problems.
+       <http://link.springer.com/article/10.1023/A:1010920819831>`_
+       Machine learning, 45(2), pp.171-186.
+
+    .. [FC2009] Ferri, Cèsar & Hernandez-Orallo, Jose & Modroiu, R. (2009). 
```

Comment:
this is not cited. should it be?

Reply:
the provost citation is missing for the OVR.